src/main/webapp/play.html

```diff
@@ -97,6 +97,14 @@
 		
 %>
 
+	//Check HLS player is finished. If finished page should be reload
+	var video = videojs('video-player').ready(function(){
+		var player = this;
+		player.on('ended', function() {
+	  		location.reload();
```

Comment:
I think reloading whole page is not a good solution for that. We may re-initialize player or something to continue playing.